README.md

```diff
@@ -1,4 +1,4 @@
-MyExpenses
+MyExpenses  [![Bountysource](https://www.bountysource.com/badge/tracker?tracker_id=267118)](https://www.bountysource.com/trackers/267118-myexpenses?utm_source=267118&utm_medium=shield&utm_campaign=TRACKER_BADGE)
```

Comment:
Yes please remove both, and I will merge.
I think Bountysource has an interesting concept, but what I did not like about it was that a bounty that was offered and then expired simply disappeared without any indication that it ever existed.